task3/app.js

```diff
@@ -0,0 +1,103 @@
+// 1. Write a JavaScript function that takes an array of numbers as input and returns
+// a new array where each element is multiplied by 2.t
+
+const func1 = (arr) => arr.map((element) => element * 2);
+
+const result1 = func1([2, 3, 4]);
+
+console.log("func1([2, 3, 4]) =", result1);
+
+// 2. Write a JavaScript function that takes an array as input and returns the length of the array.
+
+const func2 = (arr) => arr.length;
+
+const result2 = func2([2, 3, true, "a"]);
+
+console.log("func2([2, 3, true, 'a']) =", result2);
+
+// 3. Write a JavaScript function that takes an array of strings as input, sorts the strings
+// in ascending order, and returns the sorted array.
+
+const func3 = (arr) => arr.sort();
+
+const result3 = func3(["bc", "ab", "cd", "aa", "a1", "aaa"]);
+
+console.log('func3(["bc", "ab", "cd", "aa", "a1", "aaa"]) =', result3);
+
+// 4. Write a JavaScript function that takes two arrays as input, combines them into
+// a single array, and returns the result.
+
+const func4 = (arr1, arr2) => [arr1, arr2].flatMap((elem) => elem);
```

Comment:
❗  This solution is excessive. 
The operation of arrays concatenations is quite often in work. The more useful way are
```
[...arr1, ...arr2]
```
or
```
arr1.concat(arr2)
```